Sources/HaCWebsiteLib/routes.swift

```diff
@@ -44,3 +47,28 @@ public func serveWebsite() {
   Kitura.addHTTPServer(onPort: Config.listeningPort, with: getWebsiteRouter())
   Kitura.run() // This call never returns
 }
+
+func testDatabase() {
+  do {
+    let driver = try Driver(
+      masterHostname: "hac-db",
+      readReplicaHostnames: [],
+      user: "richard",
+      password: "test",
+      database: "hac"
+    )
+    Database.default = Database(driver)
+    try Pet.prepare(Database.default!)
```

Comment:
It appears here that we're setting a default drive _and_ setting it explicitly (part of the reason why we have a `!` here)

Because we're setting a default, can we omit it here?